cmd/snap/cmd_find.go

```diff
@@ -84,8 +84,22 @@ func getPriceString(prices map[string]float64, suggestedCurrency, status string)
 	return formatPrice(price, currency)
 }
 
+type SectionName string
+
+func (s *SectionName) Complete(match string) []flags.Completion {
+	// TODO cache the result
+	cli := Client()
+	sections, _ := cli.GetSections()
+	ret := make([]flags.Completion, len(sections))
+	for _, s := range sections {
+		ret = append(ret, flags.Completion{Item: s})
+	}
+	return ret
+}
+
 type cmdFind struct {
-	Private    bool `long:"private"`
+	Private    bool        `long:"private"`
+	Section    SectionName `long:"section" optional:"yes"`
```

Comment:
"optional" is the default, isn't it?  Otherwise "private" above would be bogus. Can we please drop the explicit listing if so?